File: C++/1074.cpp
#include <iostream>
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{
    int num;
    int caso;

    cin >> caso;

    for (; caso > 0; caso--)
    {

        cin >> num;
        if (num == 0)
            cout << "NULL"
                 << "\n";
        if (num % 2 != 0 && num < 0)
            cout << "ODD NEGATIVE"
                 << "\n";
        if (num % 2 == 0 && num < 0)
            cout << "EVEN NEGATIVE"
                 << "\n";
        if (num % 2 != 0 && num > 0)
            cout << "ODD POSITIVE"
                 << "\n";
        if (num % 2 == 0 && num > 0)
            cout << "EVEN POSITIVE"
                 << "\n";
    }
}

File: C#/1074.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        int caso = Convert.ToInt32(Console.ReadLine());

        for (; caso > 0; caso--)
        {
            int num = Convert.ToInt32(Console.ReadLine());
            if (num == 0)
                Console.WriteLine("NULL");
            if (num % 2 != 0 && num < 0)
               Console.WriteLine("ODD NEGATIVE");
            if (num % 2 == 0 && num < 0)
                Console.WriteLine("EVEN NEGATIVE");
            if (num % 2 != 0 && num > 0)
                Console.WriteLine("ODD POSITIVE");
            if (num % 2 == 0 && num > 0)
               Console.WriteLine("EVEN POSITIVE");
        }

    }
}